fastchat/serve/openai_api_server.py

```diff
@@ -135,15 +135,15 @@ async def check_length(request, prompt, max_tokens):
         response = await client.post(
             worker_addr + "/model_details",
             headers=headers,
-            json={},
+            json={"model": request.model},
```

Comment:
Note: I found all the spots that the OpenAI server calls the worker and ensured it includes model.  Are there other places that need this fix?

Reply:
It seems you fixed all places.